styles/components/blocks/code.scss

```diff
@@ -12,7 +12,7 @@
   pre {
     white-space: pre;
     max-width: 100%;
-    overflow: hidden;
+    overflow: auto;
     overflow-x: scroll;
     overflow-x: overlay;
```

Comment:
Seems like this component has a bunch of `overflow` properties. And, in theory, the `overflow-x` properties below should take precedence over the `overflow: auto;`.

I'd try removing those `overflow-x` properties, and keep only `overflow: auto;` or `overflow-x: auto;` to prevent vertical scrollbars to appear whitin the code block

Reply:
Good catch! I've kept only `overflow: auto;` and have removed the other two `overflow-x` properties.